spec/support/providers.rb

```diff
@@ -17,6 +17,14 @@ def stub_delayed_job
     allow(Delayed::Job).to receive(:count).and_return(1)
   end
 
+  def stub_delayed_job_where
```

Comment:
this is what was up with the dj order / where clause

Reply:
👍